### PR TITLE
[Import Group] Merge for GetPlaceStatVars API

### PR DESCRIPTION
**Pull request by shifucun**
Current and Import Group golden files all match except for a variable "NumberOfMonths_WetBulbTemperature_35COrMore_SSP245_MeanRelativeHumidity". This is newly added in the import group.

### CODE REVIEW

**Comment by Spaceenter** [comment]
It seems like @pradh is changing branch cache to import group too, so maybe need to include that changes too? I see the merge in this PR still treats branch cache separately.

File: internal/server/resource/resource.go [hunk 1]
```diff
@@ -44,6 +44,8 @@ type Metadata struct {
 	Bq               string
 	BtProject        string
 	BranchBtInstance string
```
Comment (pradh) [inline internal/server/resource/resource.go]: how is BranchTable different from BranchBtInstance?

**Review by pradh — APPROVED** [review]
Very strange... when I was reviewing earlier I saw some different diffs with version proto updates, etc.

File: internal/util/util.go [hunk 2]
```diff
@@ -270,18 +270,27 @@ func bToMb(b uint64) uint64 {
...
+		// Initialize the set map.
+		for _, s := range str {
+			if i == 0 {
+				m[s] = struct{}{}
+			} else {
+				if _, ok := m[s]; !ok {
+					result = append(result, s)
+				}
+				m[s] = struct{}{}
+			}
```
[the remark this answers is not in the document]
Comment (shifucun) [reply internal/util/util.go:290]: Realize there is a bug in initial code. Should only add the new token to the map when it does not exist. Updated.

**Comment by shifucun** [comment]
> Very strange... when I was reviewing earlier I saw some different diffs with version proto updates, etc.

Made a wrong push.. Can you review again?

**Comment by shifucun** [comment]
> It seems like @pradh is changing branch cache to import group too, so maybe need to include that changes too? I see the merge in this PR still treats branch cache separately.

For now will still treat the branch cache as a special one. Later on, will add merging config for each group then all cache can be treated in the same way.

**Comment by pradh** [comment]
> It seems like @pradh is changing branch cache to import group too, so maybe need to include that changes too? I see the merge in this PR still treats branch cache separately.

So the "branch cache" import group gets built in place of the current branch cache.

**Comment by pradh** [comment]
> > Very strange... when I was reviewing earlier I saw some different diffs with version proto updates, etc.
> 
> Made a wrong push.. Can you review again?

Yeah, LG

**Comment by shifucun** [comment]
Thank you all for review!

**Pull request merged**